// Copyright 2015 Maintainers of OarphKit
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//   http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

option optimize_for = SPEED; 
option cc_generic_services = false;

package ok_test_msg; // Put generated classes in namespace ok_test_msg
option java_package = "org.oarphkittest.msg.proto"; // Java package
option java_outer_classname = "TestMessage";

message TestMessage {
  optional bytes raw_bytes = 1;
  optional string raw_str = 2;
  repeated int64 nums = 3;
}
